facia-tool/public/js/widgets/columns/fronts-search.html

```diff
@@ -0,0 +1,31 @@
+<div class="col__inner scrollable">
+    <div class="title">
+        Search
+    </div>
+    <div class="search-form">
+        <input
+          type="text"
+          placeholder="search for fronts"
+          data-bind='
+            value: searchTerm,
+            valueUpdate: ["afterkeydown", "afterpaste"]
```

Comment:
I know that I've told you to copy from autocomplete, but can you try [textInput binding](http://knockoutjs.com/documentation/textinput-binding.html). Should work better